bash/setup_maven.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+#/*******************************************************************************
+# * Copyright (c) 2020 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Eclipse Public License 2.0 which is available at
+# * http://www.eclipse.org/legal/epl-2.0.
+# *
+# * SPDX-License-Identifier: EPL-2.0
+# *******************************************************************************/
+
+# --------------------------------------------------------------------------------------------
+# This script tries to setup the environment variables for native librarie sucha s vtk and zmq
+# --------------------------------------------------------------------------------------------
+set +e
```

Comment:
Shouldn't this be a "-" to exit the script if an error occures?

Reply:
Now it is set to -e.

Yes, this script should properly fail if the desired maven version cannot be installed to the system. Otherwise a whole workflow might still execute but with a random maven version which is part of the ci runner.